### PR TITLE
Table - scrollable responsive tables (ED-187)

### PR DESCRIPTION
**Pull request by krezka**

### CODE REVIEW

File: src/utils/table/Table.tsx [hunk 1]
```diff
@@ -6,12 +6,13 @@ import * as styles from './Table.scss';
 export const Table = ({ children, align, scrollable, className }:
       { children: any,
         align?: 'right' | 'left' | 'center',
-        scrollable?: any,
+        scrollable?: boolean | 'lg',
```
Comment (peculiarity) [inline src/utils/table/Table.tsx]: I dont like it. You combine multiple types. Scrollable<Boolean> is fine but the other one should be in separatr property maybe.  What about size ?

File: src/index.scss [hunk 2]
```diff
@@ -43,6 +44,11 @@ body {
   border-radius: 5.5px;
 }
 
+::-webkit-scrollbar-corner {
+  background-color: rgba(white, 0.25);
+  border-radius: 5.5px;
```
Comment (peculiarity) [inline src/index.scss]: 5.5px border radius ?

Comment (krezka) [reply src/index.scss]: Copied from scrollbar thumb. I'll change both

File: src/utils/table/Table.tsx (hunk 1 above)
Comment (krezka) [reply src/utils/table/Table.tsx]: I had smth in mind with 'lg' scrollable, but when tried to write explanation it turns out overcomplicated ;) I'm removing this.

**Pull request merged**